### PR TITLE
User profile icons layout changed

### PR DESCRIPTION
**Pull request by harshvitra**
User profile icons layout changed for public profiles

### CODE REVIEW

File: public/assets/images/navigation/GlobeSelected.tsx [hunk 1]
```diff
@@ -1,8 +1,9 @@
+import { propEach } from '@turf/turf';
```
Comment (norbertschuler) [inline public/assets/images/navigation/GlobeSelected.tsx:1]: What is this?

Comment (harshvitra) [reply public/assets/images/navigation/GlobeSelected.tsx:1]: My bad

File: src/features/user/UserProfile/components/UserShareAndSupport.tsx [hunk 2]
```diff
@@ -88,35 +89,56 @@ export default function UserShareAndSupport({ userprofile }: any) {
...
+            <p className={styles.bottomRowText}>{t('me:support')}</p>
           </div>
-         )}
-        <div className={styles.iconTextColumnSupport}>
+        )}
+
+        {userprofile.url && (
+          <a
+            className={styles.iconTextColumn}
+            href={userprofile.url}
```
Comment (norbertschuler) [inline src/features/user/UserProfile/components/UserShareAndSupport.tsx:112]: This does not work, e.g. http://localhost:3000/t/www.epublica.de for my test profile:
Bildschirmfoto 2020-12-04 um 13 51 47

Also this is regarded as an unsafe link!

Comment (norbertschuler) [reply src/features/user/UserProfile/components/UserShareAndSupport.tsx:112]: Happening when users do not explicitly add http:// or https:// for their websites:

Bildschirmfoto 2020-12-04 um 13 55 29

Comment (norbertschuler) [reply src/features/user/UserProfile/components/UserShareAndSupport.tsx:112]: To solve the security issue just add `rel="noopener noreferrer" `

**Pull request merged**